addons/account/migrations/13.0.1.1/openupgrade_analysis.txt

```diff
@@ -182,6 +182,7 @@ account      / account.move             / message_follower_ids (one2many): NEW r
 account      / account.move             / message_ids (one2many)        : NEW relation: mail.message
 account      / account.move             / message_main_attachment_id (many2one): NEW relation: ir.attachment
 account      / account.move             / partner_id (many2one)         : not a function anymore
+account      / account.move             / reversal_move_id (one2many)   : NEW relation: account.move
```

Comment:
It seems we finally have this field. We should populate it.

Reply:
What do you mean by populate? It's a one2many

Reply:
Oh, really, with such name?! Then nothing to do.